exercises/perfect-numbers/perfect_numbers_test.rb

```diff
@@ -0,0 +1,44 @@
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'example'
+
+class PerfectNumberTest < Minitest::Test
+  def test_aliquot_sum_zero
+    assert_equal 0, PerfectNumber.new(1).aliquot_sum
```

Comment:
Thanks Bernardo that makes sense since the core functionality should be the classify. This is my first time contributing to open source so I tried to follow the lua example very strictly. Thanks for the feedback I'll start making the changes now :).